infrastructure/terraform/README.md

```diff
@@ -100,16 +114,46 @@ installation.
+
+There are two components in this solution that requires data for proper installation and functioning. One is the Looker Studio Dashboard, you only deploy the dashboard after you have executed the last step in the Installation Guide successfully. Another is the ML pipeline, the pipelines compilation requires views and tables to be created so that it can read their schema and define which kind of transformations the columns will suffer during the pipeline execution.  
 
-Now that you have deployed all assets successfully for the first time in your Google Cloud Project, the data must flow through all components. 
+To start the data flow you must perform the following tasks:
 
-At this point in time, you have two options: a) trigger your Cloud Workflow to execute your Dataform worflow. b) wait until the next day 
-when the Cloud Workflow is going to execute according to your schedule. Next, you will need to execute the BigQuery stored procedures 
-having the prefix "invoke_backfill_*".
+1. Run the Cloud Workflow
+
+    On the Google Cloud console, navigate to Workflows page. You will see a Workflow named `dataform-prod-incremental`, then under Actions, click on the three dots and `Execute` the Workflow.
+
+1. Invoke the BigQuery stored procedures having the prefix `invoke_backfill_*` to backfill the feature store in case the GA4 Export has been enabled a long time ago before installing MDE.
+
+    On the Google Cloud console, navigate to BigQuery page. On the query composer, run the following queries to invoke the stored procedures.
+    ```sql
+    CALL `feature_store.invoke_backfill_customer_lifetime_value_label`();
+    CALL `feature_store.invoke_backfill_purchase_propensity_label`();
+    CALL `feature_store.invoke_backfill_user_dimensions`();
+    CALL `feature_store.invoke_backfill_user_lifetime_dimensions`();
+    CALL `feature_store.invoke_backfill_user_lookback_metrics`();
+    CALL `feature_store.invoke_backfill_user_rolling_window_lifetime_metrics`();
+    CALL `feature_store.invoke_backfill_user_rolling_window_metrics`();
+    CALL `feature_store.invoke_backfill_user_scoped_lifetime_metrics`();
+    CALL `feature_store.invoke_backfill_user_scoped_metrics`();
+    CALL `feature_store.invoke_backfill_user_scoped_segmentation_metrics`();
+    CALL `feature_store.invoke_backfill_user_segmentation_dimensions`();
+    CALL `feature_store.invoke_backfill_user_session_event_aggregated_metrics`();
+    ```
+
+1. Redeploy the ML pipelines using Terraform.
+
+    On your code editor, change the variable `deploy_pipelines` from `true` to `false`, on the TF variables file `${TERRAFORM_RUN_DIR}/terraform.tfvars`.
+    Next, destroy the ML pipelines component by applying the terraform configuration.
```

Comment:
Why are we doing the destroy now? Shouldn't we set it to "false" in the beginning?

Reply:
Re-phrased it to avoid confusion.